shared/actions/chat.js

```diff
@@ -575,7 +575,7 @@ function * _startNewConversation (conversationIDKey: ConversationIDKey) {
       yield put(selectConversation(newConversationIDKey, false))
     }
     // Load the inbox so we can post, we wait till this is done
-    yield call(_loadInbox, {payload: {onlyLoad: tlfName}, type: 'chat:loadInbox'})
+    yield call(_getInboxAndUnbox, {payload: {conversationIDKey: newConversationIDKey}, type: 'chat:getInboxAndUnbox'})
```

Comment:
:sheep: type the argument here like: `call(_getInboxAndUnbox, ({payload ...}: Constants.GetInboxAndUnbox))`